### DISCUSSION TITLE
Only javascript,css and html highlighting

### DISCUSSION THREAD

**Discussion by Sarfinaa** [PrismJS/prism#2985] [Q&A]
I used babel-plugin-prismjs to use other languages but its not working in react and i have to manually import all the language files i want to highligt. Is there any other way i can highlight all the languages??Please help me out.

**Comment by mAAdhaTTah** [comment]
Please share your babel configuration or a reproducible example.

**Reply by Sarfinaa** [reply]
```json
{
  "plugins": [
    ["prismjs", {
        "languages": [ "css", "markup","javascript","java"],
        "plugins": ["line-numbers"],
        "theme": "default",
        "css": true
    }]
  ]
}
```
This is my .babelrc file config.
I added java but its not highlighting the code.
And this is how i am importing prismjs
```js
import React, { useState, useEffect } from "react";
import Prism from "prismjs";
import "./Input.css";
import "prismjs/themes/prism.css";

//import 'prismjs/components/prism-java'
``` 
Whenever i uncomment this code `import 'prismjs/components/prism-java'` it highlights java but i have to highlight 10+ languages so i cant manually import each language. So please help me out sir.

**Reply by mAAdhaTTah** [reply]
That looks correct. Babel itself is probably not running over your code, so you'll need to get that working correctly with your build tool. This is probably not an issue w/ the babel plugin.

**Reply by Sarfinaa** [reply]
Do you know a way how i can solve this problem ?

**Reply by mAAdhaTTah** [reply]
Fix your build tool. The answer is going to vary widely depending on your particular setup.
